interface/frontend/src/components/report-and-export/RSNAStandardTemplate.vue

```diff
@@ -197,6 +197,9 @@ import RSNAStandardTemplateFancy from './RSNAStandardTemplateFancy'
 import OneLineParagraph from './OneLineParagraph'
 
 import Nodule from '../annotate-and-segment/Nodule'
+import html2canvas from 'html2canvas'
+import JSPDF from 'jspdf'
+import $ from 'jquery-slim'
```

Comment:
I guess due to the #215 we have to avoid usage of `jquery`, even slim version.
From the code below, I see that `jquery` was only used to get the desired DOM. 
This can easily be done through `$refs` method from [`Vue API`](https://vuejs.org/v2/api/).

Reply:
uhh had missed this one. Let me fix in a few :thumbsup: I am quite new Vue

Reply:
I am not sure `html2canvas` is a good idea since it will basically make a photo of selected element. So if you make the windows small, the result resolution will be really low. And second thing - it doesn't leave the text as text. Everything inside specified element will be taken as an image.